azure/create-cluster.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# fail if trying to reference a variable that is not set.
+set -u
+# exit immediately if a command fails
+set -e
+
+rg=${RESOURCE_GROUP_NAME}
+
+az group create -l eastus -n ${rg}
```

Comment:
Can you change this to eastus2 or make it configurable?

Reply:
It seems that eastus2 is less expensive, why do we want eastus2?

Reply:
Because we have a lot more capacity there. So, if you want to run a large cluster, you almost always need it (as my experience showed).

I also commented on this to make it configurable if possible as it used to be on ec2.